README.md

```diff
@@ -174,6 +174,26 @@ Get the `SpanID` of the `activeSpan` in the `Context`.
 go2sky.SpanID(ctx)
 ```
 
+## Application Log  Report
+
+```go
+
+r, err := reporter.NewGRPCReporter("oap-skywalking:11800")
+if err != nil {
+    log.Fatalf("new reporter error %v \n", err)
+}
+defer r.Close()
+
+xSkyapmError,xSkyapmLogger:=go2sky.NewSkyLogger(r)
+if xSkyapmError!=nil{
+    log.Fatalf("new NewSkyLogger error %v \n", xSkyapmError)
+}
+
+
+xSkyapmLogger.WriteLogWithContext(ctx,"ERROR","user has logout!")
+
```

Comment:
@arugal What do you think about this way?

Reply:
I want to check, how this works with the existing golang logger system. 
In my mind, it should be a logger wrapper, which could make users don't have to change the existing logger codes(need to change how to initialize logger). We could provide two ways, <1> output logs in both log systems. <2> SkyWalking logger overrides the original.

Reply:
> I want to check, how this works with the existing golang logger system. In my mind, it should be a logger wrapper, which could make users don't have to change the existing logger codes(need to change how to initialize logger). We could provide two ways, <1> output logs in both log systems. <2> SkyWalking logger overrides the original.

We need to develop plugin for the logging framework, like as [GRPCLogClientAppender](https://github.com/apache/skywalking-java/blob/main/apm-application-toolkit/apm-toolkit-log4j-2.x/src/main/java/org/apache/skywalking/apm/toolkit/log/log4j/v2/x/log/GRPCLogClientAppender.java)